### PR TITLE
issue #718: POST with GeographyPoint throws Object must implement

### PR DESCRIPTION
**Pull request by xuzhg**
### Issues
#718 

### Description
Make the post/update spatial property work

### CODE REVIEW

File: OData/test/E2ETest/WebStack.QA.Test.OData/Spatial/SpatialFactory.cs [hunk 1]
```diff
@@ -0,0 +1,1359 @@
...
+namespace Microsoft.Spatial
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+
+    /// <summary>
+    /// Geography Factory
+    /// </summary>
+    public static class GeographyFactory
```
Comment (xuzhg) [inline OData/test/E2ETest/WebStack.QA.Test.OData/Spatial/SpatialFactory.cs]: This class is copied from ODL.

**Comment by xuzhg** [comment]
@VikingsFan is added to the review. #Closed

**Comment by xuzhg** [comment]
@lewischeng-ms is added to the review. #Closed

**Comment by xuzhg** [comment]
@luoyan0517 is added to the review. #Closed

**Comment by VikingsFan** [comment]
:shipit:

**Comment by xuzhg** [comment]
Merged at 7916898fd45ccf7ce73c87b0c9d3c831274f6527. Thanks.

**Pull request closed without merging**